{"name": "Система вернёт ошибку, если неправильно указать Пароль при авторизации", "status": "passed", "start": 1725227708235, "stop": 1725227709319, "uuid": "ed1cc054-75c0-4e42-92ac-affd0451a1ff", "historyId": "3498956ba56b82a42f962871dc53771c", "testCaseId": "3498956ba56b82a42f962871dc53771c", "fullName": "tests.test_login_courier.TestLogiEndpoints#test_login_false_password", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLogiEndpoints"}, {"name": "host", "value": "Notebook"}, {"name": "thread", "value": "9016-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}